tests/cpp/data/test_data.cc

```diff
@@ -52,4 +53,15 @@ TEST(SparsePage, PushCSC) {
     ASSERT_EQ(inst[i].index, indices_sol[i % 3]);
   }
 }
+
+TEST(SparsePage, PushCSCAfterTranspose) {
```

Comment:
Can we have specific expected results other than not crashing in this test?

Reply:
i have modified the test. please check now...